client/dashboard/task-list/index.js

```diff
@@ -409,13 +413,19 @@ class TaskDashboard extends Component {
 }
 
 export default compose(
+	window.wcSettings.preloadOptions
```

Comment:
Could you do this with an `if` statement? It seems a little complicated for what it does.

Reply:
I originally had it like this:

```js
withOptionsHydration( {
    ...( window.wcSettings.preloadOptions || {} ),
} )
```
It was nicer to read, but I'd rather not call the hydration function if the data isn't there because resolvers in the data store will cache the empty values, preventing future api requests from going through.